Automattic/wp-calypso
# Remove sparkline from Stats menu item for Ecommerce Plan users only

### Specification

The new Ecommerce menu structure will move the parent menu item _Stats_ under _Jetpack_. This would mean that the _Stats_ menu will be a **submenu**, and the sparkline looks _weird_ in this case. This PR aims to hide the sparkline in Ecommerce plans. 

More context: peapX7-Um-p2

### Validation

1. Navigate to any page inside Calypso admin.
2. Ensure that the sparkline in the _Stats_ menu item is visible.
3. Apply this patch.
4. Visit an Ecommerce site and ensure the sparkline is removed.
5. Visit a non-Ecommerce site and ensure the sparkline is visible.
